erpnext/selling/doctype/customer/customer.js

```diff
@@ -122,6 +122,11 @@ frappe.ui.form.on("Customer", {
 					{party_type:'Customer', party:frm.doc.name});
 			});
 
+			frm.add_custom_button(__('Statement of Account'), function() {
```

Comment:
@this came from general ledger. it set Party first then party type.
I used frappe.run_serially but for timeout this gif goes into infinite loop
Should I removed this validation or should we use party_type and party link in filter instead of Multiselect

Reply:
See if we can override how the filters are set in General Ledger instead, doesn't make sense to set party before setting party type

Reply:
its came for general ledger too. there is some async problem

Reply:
You'll have to check the local state (variables) when this is being called, see if the party type exists, and it's comparing with the wrong values or something